configs/next/packages/tcmalloc/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="TCMalloc"
 ATSRC_PACKAGE_VER=2.6
-ATSRC_PACKAGE_REV=d7be93856093
+ATSRC_PACKAGE_REV=34f78a2dcd81
```

Comment:
There was a version bump to 2.6.90, a.k.a. 2.7rc according to the NEWS file...  I believe we should keep `ATSRC_PACKAGE_VER` set to 2.6, thus I'm approving this patch.